niworkflows/workflows/epi/refmap.py

```diff
@@ -0,0 +1,220 @@
+        same session, and have the same PE direction and total readout time.
+
+    Inputs to this workflow might be a list of :abbr:`SBRefs (single-band references)`,
+    a list of fieldmapping :abbr:`EPIs (echo-planar images)`, a list of
+    :abbr:`BOLD (blood-oxygen level-dependent)` images, or a list of
+    :abbr:`DWI (diffusion-weighted imaging)` datasets.
+    Please note that these different modalities should be mixed together in any
+    case for this particular workflow.
+
+    For BOLD datasets, the workflow may be set-up to execute an algorithm that determines
+    the nonsteady states in the beginning of the timeseries (also called *dummy scans*),
+    and uses those for generating a reference of the particular run, since the nonsteady
+    states are known to yield better T1 contrast (and hence perhaps better signal for
+    image registration).
+
+    Relatedly, the workflow also provides a global signal drift estimation per run.
+    This global signal drift is typically interesting for DWIs: because *b=0*
+    volumes are typically scattered throughout the scan, this drift can be
+    fit an exponential decay to model the signal drop caused by the increasing
+    temperature of the device (this is heavily related to BOLD *nonsteady states*
+    described above, as these are just the few initial instants when the exponential
+    decay is much faster).
+
+    Workflow Graph
+        .. workflow::
+            :graph2use: orig
+            :simple_form: yes
+
+            from niworkflows.workflows.epi.refmap import init_epi_reference_wf
+            wf = init_epi_reference_wf(omp_nthreads=1)
+
+    Parameters
+    ----------
+    omp_nthreads : :obj:`int`
+        Maximum number of threads an individual process may use
+    name : :obj:`str`
+        Name of workflow (default: ``epi_reference_wf``)
+    auto_bold_nss : :obj:`bool`
+        If ``True``, determines nonsteady states in the beginning of the timeseries
+        and selects them for the averaging of each run.
+        IMPORTANT: this option applies only to BOLD EPIs.
+
+    Inputs
+    ------
+    in_files : :obj:`list` of :obj:`str`
+        List of paths of the input EPI images from which reference volumes will be
+        selected, aligned and averaged.
+
+    Outputs
+    -------
+    epiref : :obj:`str`
+        Path of the generated EPI reference file.
+    xfms : :obj:`list` of :obj:`str`
+        List of rigid-body transforms in LTA format to resample from
+        the reference volume of each run into the ``epiref`` reference.
+    volumes : :obj:`list` of :obj:`str`
+        List of paths to the reference volume generated per input run.
+    gs_drift : :obj:`list` of :obj:`list` of :obj:`float`
+        A list of global signal drift factors for the set of volumes selected
+        for averaging, per run.
+    n_dummy : :obj:`list` of :obj:`int`
+        Number of nonsteady states at the beginning of each run (only BOLD with
+        ``auto_bold_nss=True``)
```

Comment:
I'm not sure we have conventions here, but I think we can be a bit clearer. Verbosity makes it less likely we have to return to the docs to know what we're connecting. For files using `_file` and `_files` is very common in nipype. I don't think `gs` adds much.

```suggestion
    epi_ref_file : :obj:`str`
        Path of the generated EPI reference file.
    xfm_files : :obj:`list` of :obj:`str`
        List of rigid-body transforms in LTA format to resample from
        the reference volume of each run into the ``epiref`` reference.
    per_run_ref_files : :obj:`list` of :obj:`str`
        List of paths to the reference volume generated per input run.
    drift_factors : :obj:`list` of :obj:`list` of :obj:`float`
        A list of global signal drift factors for the set of volumes selected
        for averaging, per run.
    n_dummy_scans : :obj:`list` of :obj:`int`
        Number of nonsteady states at the beginning of each run (only BOLD with
        ``auto_bold_nss=True``)
```

These are just suggestions, and happy to consider alternatives.